examples/keyvalue/server/observer.go

```diff
@@ -32,7 +32,7 @@ type Observer struct {
 
 // OnInit is called during service init process. Returning an error halts the init?
 func (o *Observer) OnInit(svc service.Host) error {
-	svc.Logger().Info(
+	svc.Logger().Sugar().Infow(
```

Comment:
`Sugar()` is low enough overhead/memoized that we don't need a `svc.SugaredLogger()`? Cool!